Resources/Prototypes/_NF/Entities/Objects/Misc/rubber_stamp.yml

```
@@ -81,3 +81,17 @@
   - type: Sprite
     sprite: _NF/Objects/Misc/stamps.rsi
     state: stamp-pal
+        
+- type: entity
+  name: Director of Care's rubber stamp
+  parent: RubberStampBase
+  id: RubberStampDoc
+  suffix: DO NOT MAP
+  components:
+  - type: Stamp
+    stampedName: stamp-component-stamped-name-doc
+    stampedColor: "#5b97bc"
+    stampState: "paper_stamp-nf-doctor"
```

```suggestion
    stampState: "paper_stamp-cmo"
```
I'd use this to be visually distinct from the doctor's stamp.